_includes/single-main-grid.html

```diff
@@ -1,9 +1,9 @@
 <div class="c-grid-item">
   <div class="c-grid-item__container">
-    <a href="#" class="c-grid-item__link" aria-label="Visit First Officers page">
-      <img class="c-grid-item__image" src="assets/homepage-first-officer-2.png" alt="An image of a First Officer">
+    <a href="#" class="c-grid-item__link" aria-label="Visit {{card.position}} page">
+      <img class="c-grid-item__image" src="{{card.image}}" alt="An image of a First Officer">
```

Comment:
Looks like this `alt` text was missed when you were making things dynamic.

_However,_ I probably would hide these images from assistive technologies altogether. This is because they don't really provide useful information or content. You can do this with a blank alt attribute: `alt=""`

https://davidwalsh.name/accessibility-tip-empty-alt-attributes